nuxt.config.ts

```diff
@@ -196,29 +211,16 @@ const config: NuxtConfig = {
     "@nuxtjs/i18n",
     "@nuxtjs/redirect-module",
     "@nuxtjs/sentry",
-    "@nuxtjs/sitemap",
     "cookie-universal-nuxt",
+    "@nuxtjs/sitemap",
```

Comment:
Is the order significant?

Reply:
The sitemap extension is reccommend to be last so it can capture any routes that might be generated or modified by other plugins.

We should probably add a comment to explain why it should generally be last